cartridges/LINK_adyen/cartridge/controllers/CheckoutServices.js

```diff
@@ -6,37 +6,28 @@ server.extend(module.superModule);
 var COHelpers = require('*/cartridge/scripts/checkout/checkoutHelpers');
 var adyenHelpers = require('*/cartridge/scripts/checkout/adyenHelpers');
 
-function getEncryptedData() {
-  var paymentForm = server.forms.getForm('billing');
-  return paymentForm.creditCardFields.adyenEncryptedData.value;
-}
-
 server.append('SubmitPayment',
   server.middleware.https,
   function (req, res, next) {
     var viewData = res.getViewData();
-    viewData.adyenEncryptedData = getEncryptedData();
     var paymentForm = server.forms.getForm('billing');
 
-    viewData.paymentInformation = {
-      cardType: {
-        value: paymentForm.creditCardFields.cardType.value
-      },
-      cardNumber: {
-        value: paymentForm.creditCardFields.cardNumber.value
-      },
-      expirationMonth: {
-        value: parseInt(paymentForm.creditCardFields.expirationMonth.selectedOption, 10)
-      },
-      expirationYear: {
-        value: parseInt(paymentForm.creditCardFields.expirationYear.value, 10)
-      }
-    };
-
+      viewData.adyenEncryptedCardNumber = paymentForm.creditCardFields.adyenEncryptedCardNumber.value;
+      viewData.adyenEncryptedExpiryMonth = paymentForm.creditCardFields.adyenEncryptedExpiryMonth.value;
+      viewData.adyenEncryptedExpiryYear = paymentForm.creditCardFields.adyenEncryptedExpiryYear.value;
+      viewData.adyenEncryptedSecurityCode = paymentForm.creditCardFields.adyenEncryptedSecurityCode.value;
+
+      viewData.paymentInformation = {
+            cardType: {
+              value: paymentForm.creditCardFields.cardType.value
+            },
+            cardNumber: {
+              value: paymentForm.creditCardFields.cardNumber.value
```

Comment:
is this the masked cardNumber ?

Reply:
Yes, this is only the BIN. 
Will create masked card number once LastFour is available in callback.